spec/support/aws_kms_client.rb

```diff
@@ -25,6 +25,7 @@ def stub_mapped_aws_kms_client(configs)
       config = configs.find do |c|
         c[:ciphertext] == context.params[:ciphertext_blob]
       end
+      #binding.pry
```

Comment:
i am so sorry you had to debug this complicated stub that i wrote 😭 
but let's not leave this in
```suggestion
```

Reply:
fwiw what i learned here is that it does, in fact, work